includes/Core/Admin/Screen.php

```diff
@@ -243,18 +247,15 @@ public function enqueue_assets( Assets $assets ) {
 	 * @param Context $context Plugin context.
 	 */
 	private function render( Context $context ) {
-		if ( ! $this->args['render_callback'] ) {
-			return;
-		}
-
-		?>
-		<div class="googlesitekit-plugin">
-			<?php
-				$this->render_noscript_html();
-
-				call_user_func( $this->args['render_callback'], $context );
-			?>
-		</div>
-		<?php
+		$cb = isset( $this->args['render_callback'] ) && is_callable( $this->args['render_callback'] )
+			? $this->args['render_callback']
+			: function() {
+				echo '<div id="js-', esc_attr( $this->slug ), '" class="googlesitekit-page"></div>';
```

Comment:
This of course works but looks broken in my editor. How about a `printf` instead?

<img width="730" alt="image" src="https://github.com/google/site-kit-wp/assets/1621608/9a2be37d-f331-42dd-8e07-b15bd905d05d">

<img width="750" alt="image" src="https://github.com/google/site-kit-wp/assets/1621608/ae1282a7-4637-4f95-89d5-1c17da1ac734">

```suggestion
				printf( '<div id="js-%s" class="googlesitekit-page"></div>', esc_attr( $this->slug ) );
```

Reply:
No problem, updated.